prospero-common/src/main/resources/prospero-known-combinations.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "wildfly"
```

Comment:
IIUC this is still not "provisionable" without providing some channel via `--channel` argument?

Reply:
Yes the channel has to be provided separately